cloudbuild.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
+  # FRONTEND STEPS
+  # Install node packages
+  - name: 'gcr.io/cloud-builders/npm'
+    args: [ 'install' ]
+
+  # Build settings.json file from Subsitution variable _SETTINGS_JSON which contains
+  # the environment's values for the settings.json file.
+  - name: 'ubuntu'
```

Comment:
It would be nice to keep all the cloud build bits in one place.  If this is just splatting a file from an environment variable is there a reason we can't get away with a one liner here?  eg

name: 'bash'
args: [ '-c', '[[ -z "$_SETTINGS_JSON" ]] && exit 1; echo $_SETTINGS_JSON > ./settings.json' ]

or similar?

Reply:
That makes sense, I can attempt that and update if I am able to make the change or comment here if I run into issues with it.

Reply:
For better documentation in build output I kept basically the same file structure, but it is all encapsulated in the cloudbuild.yaml file now.